src/components/LanguageSelector.jsx

```diff
@@ -4,6 +4,12 @@ import MenuItem from 'material-ui/MenuItem';
 import SelectField from 'material-ui/SelectField';
 
 const LanguageSelector = ({ languages, onChange, selected }) => {
+  const styles = {
```

Comment:
I agree with moving the in-line styling into the css folder. But on the issue of width between tokens, wont there leave a pretty big amount of empty space between the last token and the length of the panel? Once we get more tokens, I believe then would be a better time to decrease the width between tokens.